src/Codeception/Test/Feature/CodeCoverage.php

```diff
@@ -26,8 +26,20 @@ public function codeCoverageEnd($status, $time)
             return;
         }
 
+        if (method_exists($this, 'getLinesToBeCovered')) {
```

Comment:
I thought that this change does something bigger than just supporting `@covers` and `@uses` when I wrote that comment, but I was wrong.
So, because Cept and Gherkin formats don't support `@covers` and `@uses` annotations, we don't need to add anything to format files, right?